### PR TITLE
impr(quotes): add English quotes from Kingdom Hearts (@herfave)

### PR DESCRIPTION
**Pull request by herfave**
### Description

Add English quotes from the video game series, Kingdom Hearts

### Checks

- [x] Adding quotes?
  - [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [x] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username prefixed with @ inside parentheses at the end of the PR title.

Closes #

### CODE REVIEW

**Comment by Miodec** [comment]
Please resolve conflict

**Comment by herfave** [comment]
> Please resolve conflict

Merge conflict resolved with 583e4a8b48eb5b4b03e89743ca3e392b9f73cc90

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38511,6 +38511,30 @@
...
+      "id": 7620
+    },
+    {
+      "text": "A scattered dream that's like a far-off memory. A far-off memory that's like a scattered dream. I want to line the pieces up - yours and mine.",
+      "source": "Kingdom Hearts II",
+      "length": 142,
+      "id": 7621
+    },
+    {
+      "text": "That dream again... I've been having the same dream over and over again for as long as I can remember. An unfamiliar world filled with unfamiliar faces and unfamiliar good-byes... As if I'm experiencing a life that belonged to someone else. This island surrounded by smaller desolate isles, a vast ocean, and a never-ending sky—this is my entire world. But maybe there's something beyond these waters. Maybe there are worlds like the ones I dream of. If I could get to the outside world, would I be able to meet my friends from my dreams? I can't get the thought out of my head as I lie on the sand like a starfish day after day, feeling like I'm falling through the sky.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38528]: Missing spaces around `-` in `never-ending sky - this is my entire world.`

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**